pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -315,3 +315,8 @@ type PodInfoList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []PodInfo `json:"items"`
 }
+
+type KillerSpec struct {
+	// syscalls to kill
```

Comment:
How about `description: syscalls where killer is executed in`?

AFAIU, these are the syscalls where we will load the killer and do the check on whether to "kill" the process or not.